test/contrib/pnio_rpc.uts

```diff
@@ -204,6 +204,30 @@ p == IODWriteMultipleRes(ARUUID='01234567-89ab-cdef-0123-456789abcdef', recordDa
     ]) / conf.padding_layer(b'\xef')
 
 
+####################################################################
+####################################################################
+
++ Check I&M
+
+= IM0Block default values
+bytes(IM0Block()) == bytearray.fromhex('002000380100000000000000000000000000000000000000000000000000000000000000000000000000000000005600000000000000000001010000')
```

Comment:
Call you use `raw()` instead of `bytes()` in your tests?

Reply:
I can but the rest of the tests in this file use `bytes()`. Should I do a commit to convert the others before adding my new changes as a follow up?

Reply:
No. This is a fine as it is. Thanks for the fix!